content/source/docs/enterprise/users-teams-organizations/teams.html.md

```diff
@@ -87,4 +84,19 @@ A team can be given read, write, or admin permissions on one or more workspaces.
 - Use any workspace's "Access" tab to manage team permissions on that workspace. For full instructions, see [Managing Access to Workspaces](../workspaces/access.html).
 - For detailed information about the available permissions levels, see [Permissions](./permissions.html).
 
-When determining whether a user can take an action on a resource, TFE uses the highest permission level from that user's teams. (For example, if a user belongs to a team with read permissions on a workspace and another team with admin permissions on that workspace, that user has admin permissions.)
+When determining whether a user can take an action on a resource, TFE uses the highest permission level from that user's teams. For example, if a user belongs to a team with read permissions on a workspace and another team with admin permissions on that workspace, that user has admin permissions.
+
+Organization-level permissions (see [Managing Organization Access](./teams.html#managing-organization-access), below) can also supersede lower workspace permissions. For example, if a user belongs to a team with read permissions on a workspace but also has workspace management enabled, that user has admin permissions on the workspace. Conversely, if the team is allowed to manage the organization's Sentinel policies (which gives read access to all workspaces for enforcing policy sets) and has admin access on the workspace, the higher admin permission level is granted to the workspace.
+
+## Managing Organization Access
+
+-> **API:** See the [Teams API](../api/teams.html). <br/>
```

Comment:
The `<br>` is only necessary if there's an equivalent link to the TFE provider right below the API link. (...which I'm assuming there will be eventually, but if it's not there the callout box might look weird.)